ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -72,7 +73,7 @@
 
 (defmulti update->instruction
   "Returns the granule bulk update instruction for a single update item"
-  (fn [event-type item]
+  (fn [event-type _item]
```

Comment:
whats the significance here?

Reply:
Clojure style-guide preferred method for marking an input as not used, alternatively just an underscore.

Reply:
Ahh, I see. Good to know, will use this in the future.